fdbserver/include/fdbserver/BlobGranuleServerCommon.actor.h

```diff
@@ -140,9 +140,27 @@ struct BGTenantMap {
 	Future<Void> collection;
 };
 
+// Defines granule info that interests full restore
+struct BlobGranuleVersion {
```

Comment:
If this is only for full restore, should that be part of the name? i.e. BlobGranuleRestoreVersion?

Reply:
it sounds like a better name. fixed